docs/_documentations/mdt-intellij-getting-started.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: docs
+title: Installing Codewind on IntelliJ
+description: Installing Codewind on IntelliJ
+keywords: install, installing, IntelliJ, Eclipse, Codewind, IDE, plugin, plug-in, settings, creating, project, projects, template, code change, edit, edits, application, removing
+duration: 1 minute
+permalink: mdt-intellij-getting-started
+type: document
+---
+
+# Getting started with Codewind on IntelliJ
+Use Codewind on IntelliJ to develop your applications.
+
+## Prerequisites
+- Install [IntelliJ](https://www.jetbrains.com/idea/download/#section=mac).
+- Install Docker.
+- If you use Linux, also install Docker Compose.
+
+## Installing Codewind on IntelliJ
+1. Go to the [Eclipse Downloads page](https://download.eclipse.org/codewind/?d) for Codewind. Click **milestone** and **0.9.0**. Then, click **codewind-intellij-0.9.0.zip** to download the file.
+2. From IntelliJ, open either **Preferences** or **Settings**. A new window appears.
+   - On macOS, go to **Main**>**Preferences**.
+   - On Windows, go to **File**>**Settings**.
+3. Open **Plugins** and click the **Manage plugin repositories** icon. From the menu, click **Install Plugin from Disk...**. A window appears that displays the files on your computer.
+4. Choose the `codewind-intellij-0.9.0.zip` file and click **Open**.
+
+## Creating your first project from a template
+1. Start IntelliJ. From the welcome page, click **Create New Project**. The **New Project** window opens.
+2. Create a Codewind project by clicking **Codewind**.
+   - If Codewind isn't installed yet, click **Install Codewind** to complete the installation.
+   - If Codewind isn't running yet, click **Start Codewind**. A **Starting Codewind** window appears.
+3. After Codewind is installed and started, click **Next**. A list of available templates appears.
+   - Currently, IntelliJ supports only the Java language templates.
+4. Click the template of your choice. Then, click **Next**.
+5. Enter the **Project name:** field to name your project and enter the **Project location:** field to choose where to put it.
+6. Click **Finish**. A window for your project appears.
+7. IntelliJ detects a `pom.xml` file and asks if you want to add the file as a Maven project. In the **Non-managed pom.xml file found:** window, click **Add as Maven Project** to access your project source files.
+   - The project doesn't work if you disable the notification.
```

Comment:
Probably better to say that the project structure won't be set up properly if you don't click **Add as Maven Project**.  Also should add that the popup may disappear after a time, and if that happens, you should select the `pom.xml` file in the IntelliJ Project view, and, from the context menu, select **Add as Maven Project** to ensure the project structure is set up.